### DISCUSSION TITLE
Last word of the each heading function in editor is higher than expected

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#20618] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/86934170/194706187-346178c6-94b1-432b-a000-207d103d1a5e.png)
![image](https://user-images.githubusercontent.com/86934170/194706236-dd415057-580a-49a8-b81a-6839bf6e1e97.png)
![image](https://user-images.githubusercontent.com/86934170/194706242-3c57fb61-c3e4-4a87-a75e-f47a4f188f98.png)

For Slavic languages.

**Comment by smoogipoo** [comment, marked as answer]
Duplicate: https://github.com/ppy/osu/issues/19686
